Pipfile

```diff
@@ -15,6 +15,8 @@ PyYAML = "~=5.3.1"
 "discord.py" = {extras = ["voice"], version = "~=1.5.1"}
 async-rediscache = {extras = ["fakeredis"], version = "~=0.1.4"}
 emojis = "~=0.6.0"
+requests = "~=2.25.1"
+pre-commit = "*"
```

Comment:
I don't think either of these changes to the Pipfile are needed, could you please revert these changes?

Reply:
sure!